docker-compose-orthanc.yml

```diff
@@ -4,6 +4,11 @@ services:
     secrets:
       - orthanc.json # CAUTION: This is an experimental implementation, not for production use.
     image: mekomsolutions/orthanc:latest
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.orthanc.loadbalancer.server.port=8080"
+      - "traefik.http.routers.orthanc.rule=Host(`${ORTHANC_HOSTNAME}`)"
+      - "traefik.http.routers.orthanc.middlewares=orthanc"
```

Comment:
I am not sure how this works @corneliouzbett can you please review this?

Reply:
Probably, you need to change the port the one exposed by Orthanc Docker.

Reply:
Done